app/assets/javascripts/webauthn.js

```diff
@@ -102,6 +102,47 @@
     });
   });
 
+  $(function() {
+    var sessionForm = $(".js-webauthn-session-cli--form");
+    var sessionSubmit = $(".js-webauthn-session-cli--submit");
+    var sessionError = $(".js-webauthn-session-cli--error");
```

Comment:
It's hard to differentiate the difference between this handler vs the sessions handler if someone doesn't know to look at the jquery selectors. Could the names of these variables be renamed or assigning a name to the function?